core/src/main/java/org/apache/carbondata/scan/filter/FilterExpressionProcessor.java

```diff
@@ -286,6 +289,13 @@ private FilterResolverIntf getFilterResolverBasedOnExpressionType(
               return new RowLevelFilterResolverImpl(expression, isExpressionResolve, true,
                   tableIdentifier);
             }
+            if (currentCondExpression.getFilterExpressionType() == ExpressionType.CONTAINS
```

Comment:
For dictionary column do we need to create row level expression?? I think for dictionary column creating a include filter for like query will  good enough, because we have the dictionary values we can search in dictionary to get all the valid values and we can apply filter. Please correct me if i am wrong:)